assets_vfsdata.go

```diff
@@ -0,0 +1,211 @@
+			uncompressedSize:  1069,
+			modTime:           mustUnmarshalTextTime("2015-04-25T15:26:39-07:00"),
+		},
+	}
+
+	fs["/"].(*_vfsgen_dir).entries = []os.FileInfo{
+		fs["/repo.html.tmpl"].(os.FileInfo),
+		fs["/script.js"].(os.FileInfo),
+		fs["/style.css"].(os.FileInfo),
+	}
+
+	return fs
+}()
+
+type _vfsgen_fs map[string]interface{}
+
+func (fs _vfsgen_fs) Open(path string) (http.File, error) {
+	// TODO: Maybe clean path?
+	f, ok := fs[path]
+	if !ok {
+		return nil, os.ErrNotExist
+	}
+
+	switch f := f.(type) {
+	case *_vfsgen_compressedFile:
+		gr, err := gzip.NewReader(bytes.NewReader(f.compressedContent))
+		if err != nil {
+			// This should never happen because we generate the gzip bytes such that they are always valid.
+			panic("unexpected error reading own gzip compressed bytes: " + err.Error())
+		}
+		return &_vfsgen_compressedFileInstance{
+			_vfsgen_compressedFile: f,
+			gr: gr,
+		}, nil
+	case *_vfsgen_dir:
+		return &_vfsgen_dirInstance{
+			_vfsgen_dir: f,
+		}, nil
+	default:
+		// This should never happen because we generate only the above types.
+		panic(fmt.Sprintf("unexpected type %T", f))
+	}
+}
+
+func mustUnmarshalTextTime(text string) time.Time {
```

Comment:
I forgot the `_vfsgen_` prefix here, it should be added for consistency and to avoid possibly clash with user's own code.

I'll rename it to `_vfsgen_mustUnmarshalTextTime `.